src/utils/date.js

```
@@ -0,0 +1,46 @@
+import { format } from "date-fns";
+
+export function castDate(timestamp) {
+  return timestamp && timestamp.seconds
+    ? new Date(timestamp.seconds * 1000)
+    : timestamp;
```

shouldn't it throw instead of returning the timestamp?

---

I wanted it to have the affordance of passing through potentially malformed data, already converted values or undefined values. Throwing an error would mean we have to explicitly handle that case, showing errors or discuss/design a flow to handle that. 
The date pickers have the affordance to work even wen supplying incorrect dates, they just don't display it. Therefore I felt granting that affordance here as well made sense.